sympy/codegen/ast.py

```diff
@@ -151,7 +193,7 @@ def aug_assign(lhs, op, rhs):
         subclass these types are also supported.
 
     op : str
-        Operator (+, -, /, *, %).
+        Operator (+, -, /, \*, %).
```

Comment:
Why did you add this? 

Also, this sort of change should be failing the tests. I guess my use of `python -We:invalid` in test_travis.sh isn't working. Probably because the files are bytecompiled already when sympy is installed. I wonder if adding `-B` as suggested [here](https://github.com/sympy/sympy/pull/12028) would fix it.